articles/quickstart/native/react-native-expo/00-login.md

```diff
@@ -96,10 +101,10 @@ These values are found in the Expo config file at `app.json` or `app.config.js`.
 {
     ...
     "android": {
-      "package": "com.auth0samples"
+      "package": "YOUR_ANDROID_PACKAGE_NAME_IS_FOUND_HERE"
```

Comment:
Given that above (L93 and L95) we use `com.auth0samples` as the example value, should we keep it here? Or change these above? Having two different ways to refer to the same thing in a single tutorial can be confusing, as a developer might think these refer to different things.